### ISSUE TITLE
Cannot install on Windows

### ISSUE THREAD

**Issue by mrchief** [tcort/wkhtmltox#4]
`yarn add wkhtmltox`

gives this error and quits:

```
error modern-syslog@1.1.4: The platform "win32" is incompatible with this module.
```

**Comment by tcort** [comment]
Thanks for the report! It looks like a dependency of one of the dependencies, a logging module, isn't compatible with Microsoft Windows. I'll look into eliminating that dependency.

**Comment by tcort** [comment]
I've published version `1.0.10` which doesn't have `modern-syslog` in it's dependency chain. I don't have access to a Windows machine, so I can't test if it works on Windows, but at least that installation road block is removed.
